src/modules/claim/resolvers/credential.resolver.ts

```diff
@@ -1,17 +1,31 @@
 import {
   CredentialResolver,
+  RoleEIP191JWT,
   VerifiableCredential,
 } from '@energyweb/vc-verification';
 import { RoleCredentialSubject } from '@energyweb/credential-governance';
+import jsonwebtoken from 'jsonwebtoken';
 import { ClaimService } from '../services';
+import { IPublicClaim } from '@ew-did-registry/claims';
+import { ClaimData } from '@energyweb/vc-verification/dist/src/models';
 
 export class RoleCredentialResolver implements CredentialResolver {
   constructor(private readonly claimService: ClaimService) {}
-
   async getCredential(
+    did: string,
+    namespace: string
+  ): Promise<
+    VerifiableCredential<RoleCredentialSubject> | RoleEIP191JWT | null
+  > {
+    const vc = await this.getVerifiableCredential(did, namespace);
+    const eip191Jwt = await this.getEIP191JWT(did, namespace);
```

Comment:
Maybe we can optimize it a little bit. We need vc or jwt, so why fetch both?

Reply:
Yes, make sense